find_changed_files.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Find changed files in a build either via pull request API or compare API
+echo "event path: $GITHUB_EVENT_PATH"
+cat $GITHUB_EVENT_PATH
+COMPARE=$(jq '.compare' $GITHUB_EVENT_PATH)
+PR=$(jq -r '.pull_request._links.self.href' $GITHUB_EVENT_PATH)
+echo "-----"
+echo "compare: |$COMPARE|"
+echo "PR: |$PR|"
+echo "------"
+if [[ $COMPARE != null ]]
+then
+    echo "compare url is set, using it from the API"
+    # api.github.com/repos
+    COMPARE_API=$(echo $COMPARE | sed 's/github.com\//api.github.com\/repos\//g'| sed 's/"//g')
```

Comment:
Add a TODO: for pagination?

Reply:
oh actually i should add a page size too